src/common/systems/loading/loadgoods.cpp

```diff
@@ -97,6 +98,19 @@ bool GoodLoader::LoadValue(const Hjson::Value& values, entt::entity entity) {
         double t = values["energy"];
         universe.emplace<cqspc::Energy>(entity, t);
     }
+    if (values["consumption"].defined()) {
+        const Hjson::Value& consumption = values["consumption"];
+        double a = consumption["a"].to_double();
```

Comment:
What's a and b supposed to represent?

If it's a variable, do name out the variable instead of putting the extremely vague 'a' and 'v'